metagraph/plugins/scipy/algorithms.py

```diff
@@ -79,6 +94,21 @@ def ss_breadth_first_search_iter(
         bfs_ordered_nodes = graph.edges.node_list[bfs_ordered_incides]
         return NumpyVector(bfs_ordered_nodes)
 
+    @concrete_algorithm("flow.max_flow")
+    def ss_max_flow(
+        graph: ScipyGraph, source_node: NodeID, target_node: NodeID,
+    ) -> Tuple[float, ScipyGraph]:
+        print(f"graph.edges.value.toarray() {repr(graph.edges.value.toarray())}")
```

Comment:
Remove this `print` call

Reply:
https://github.com/metagraph-dev/metagraph/pull/117/commits/5292e5af7f6cd1201d3140afda7bc1e3cd3496ce removes the print.